database/models/index.js

```diff
@@ -0,0 +1,33 @@
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+const sequelize = new Sequelize('pcprepkit', 'pcprepkit', 'pcprepkit', {
+    host: 'localhost',
+    dialect: 'mysql',
+	define: {
+	    timestamps: false // true by default
+	}    
+});
+const db = {};
+ 
+fs
+    .readdirSync(__dirname)
+    .filter(function(file) {
+        return (file.indexOf(".") !== 0) && (file !== "index.js") && (file.indexOf("~") === -1)
+    })
+    .forEach(function(file) {
+	console.log(file);
+        var model = sequelize.import(path.join(__dirname, file));
+        db[model.name] = model;
+    });
+ 
+Object.keys(db).forEach(function(modelName) {
+    if ("associate" in db[modelName]) {
+        db[modelName].associate(db);
+    }
+});
+
+db.sequelize = sequelize;
+db.Sequelize = Sequelize;
```

Comment:
Yes, both are required. One of them is an instance of the database setup and the other is the instance of the library.